From d90111ffe23f6836ffba866dc7f9fbfc84c3cd70 Mon Sep 17 00:00:00 2001
From: "w.stott" <w.stott@engineeredarts.co.uk>
Date: Thu, 19 Aug 2021 12:43:40 +0100
Subject: [PATCH] Use yocto protobuf and ffmpeg

---
 WORKSPACE | 13 +++++++++----
 1 file changed, 9 insertions(+), 4 deletions(-)

diff --git a/WORKSPACE b/WORKSPACE
index cbbb575..9fd27f5 100644
--- a/WORKSPACE
+++ b/WORKSPACE
@@ -151,11 +151,16 @@ http_archive(
     sha256 = "a79d19dcdf9139fa4b81206e318e33d245c4c9da1ffed21c87288ed4380426f9",
     strip_prefix = "protobuf-3.11.4",
     urls = ["https://github.com/protocolbuffers/protobuf/archive/v3.11.4.tar.gz"],
-    patches = [
-        "@//third_party:com_google_protobuf_fixes.diff"
+    build_file = "@//:protobuf_yocto.BUILD",
+    patch_cmds = [
+        "mkdir -p yocto/include/google yocto/native",
+        "cp %%WORKDIR%%/recipe-sysroot/usr/lib/libprotobuf*.so yocto/",
+        "cp -r %%WORKDIR%%/recipe-sysroot/usr/include/google ./yocto/include/google",
+        "cp %%WORKDIR%%/recipe-sysroot-native/usr/bin/protoc ./yocto/native/protoc-native",
+        "cp %%WORKDIR%%/recipe-sysroot-native/usr/lib/libproto* ./yocto/native/",
     ],
-    patch_args = [
-        "-p1",
+    patches = [
+        "@//:com_google_protobuf_use_protoc_on_path.diffforbazeltoapply",
     ],
 )
 
