From 71e1fc302b41523aa4852ddf780cb2a42cf03f20 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Tue, 8 Dec 2020 17:11:48 +0800
Subject: [PATCH] use local bazel to workaround bazel paralle issue

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 configure.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.py b/configure.py
index b4907775d93..2a85871e52e 100644
--- a/configure.py
+++ b/configure.py
@@ -480,7 +480,7 @@ def check_bazel_version(min_version, max_version):
     sys.exit(1)
 
   stderr = open(os.devnull, 'wb')
-  curr_version = run_shell(['bazel', '--version'],
+  curr_version = run_shell(['../bazel/bazel', '--version'],
                            allow_non_zero=True,
                            stderr=stderr)
   if curr_version.startswith('bazel '):
-- 
2.18.2

